JotunnLib/Managers/SkillManager.cs

```diff
@@ -146,7 +114,13 @@ public Skills.SkillDef GetSkill(Skills.SkillType skillType)
         {
             if (Skills.ContainsKey(skillType))
             {
-                return Skills[skillType];
+                return new Skills.SkillDef()
+                {
+                    m_description = Skills[skillType].Description,
+                    m_icon = Skills[skillType].Icon,
+                    m_increseStep = Skills[skillType].IncreaseStep,
+                    m_skill = Skills[skillType].UID
+                };
```

Comment:
Rather than having this logic in multiple places, should we instead have a `ToSkillDef()` function in our `SkillConfig`? Might be a bit cleaner perhaps?

Reply:
Ahh yes 👍🏽, classes are awesome and we can definitely have a method like that :), I'll push some changes shortly!